README.md

```diff
@@ -112,6 +109,23 @@ and then start the `delayed_job` daemon:
 
 ## Running Specs (Automated Tests)
 
+Using Docker, you can
+
+```sh
+$> docker-compose run --rm app /bin/bash
+```
+
+To drop into a container with everything installed, then you can
+
+```sh
+$ /code> rspec
+$ /code> rspec ./spec/features
```

Comment:
When trying to run tests in docker, I am getting:

```
ActiveRecord::NoDatabaseError:
  FATAL:  database "ox_accounts_test" does not exist
```

is this something that should happen automatically?

Reply:
No, I forgot some stuff in readme, adding now